fix/anko.go

```diff
@@ -30,12 +30,13 @@ func Anko(content string) (string, error) {
 				waitParen := false
 				for {
 					if strings.HasPrefix(tl, "})") {
-						line = "}"
+						line = tl[2:]
```

Comment:
in the case that tl is "})" then tl[2:] will be "". I think you were going for `"}" + tl[2:]`